### PR TITLE
Update the awsecscontainereeceiver README to add more metadata for the instruction

### PR DESCRIPTION
**Pull request by JohnWu20**
**Description:** <Describe what has changed.>
Because we add more metadata for containerReceiver, update the README

### CODE REVIEW

File: receiver/awsecscontainermetricsreceiver/README.md [hunk 1]
```diff
@@ -144,9 +144,20 @@ aws.ecs.task.arn     | aws.ecs.task.arn
...
+aws.ecs.task.pull_started_at | aws.ecs.container.started_at
+aws.ecs.task.pull_stopped_at | aws.ecs.container.finished_at
+aws.ecs.task.known_status | ws.ecs.container.know_status
+aws.ecs.task.launch_type | aws.ecs.task.launch_type
+&nbsp; | ws.ecs.container.created_at
 &nbsp; | container.name
 &nbsp; | container.id
 &nbsp; | aws.ecs.docker.name 
+&nbsp; | aws.ecs.container.image.tag
+&nbsp; | aws.ecs.container.image.id
```
Comment (hossain-rayhan) [inline receiver/awsecscontainermetricsreceiver/README.md:159]: This two are not ECS specific. They are generic. Make them `container.image.tag` and `container.image.id`. Also, update the code as well. Use the `convention` package if possible.

**Review by hossain-rayhan — CHANGES_REQUESTED** [review]
Are we missing `container.image` in the README? Please check the comment.

File: receiver/awsecscontainermetricsreceiver/README.md (hunk 1 above)
Comment (JohnWu20) [reply receiver/awsecscontainermetricsreceiver/README.md]: By the way, there is no `exist container.image.id`

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@hossain-rayhan is it ok to merge this?

**Comment by hossain-rayhan** [comment]
> @hossain-rayhan is it ok to merge this?

Let's merge this one. I asked for a different change. We will send a different PR for that. Thanks @bogdandrutu

**Pull request merged**